### DISCUSSION TITLE
Why CURLOPT_LOW_SPEED_LIMIT is set to 1250 by default in 7.78.0

### DISCUSSION THREAD

**Discussion by CHENZENGHU** [curl/curl#7565] [Q&A] [answered]
From the manual, CURLOPT_LOW_SPEED_LIMIT is 0 by default. 
But I see in V7.78.0, it seems to be set as 1250 bytes/sec by default and the CURLOPT_LOW_SPEED_TIME is set to be 30 seconds.
Because I did not set these two variables and in my test, if I cut off the ethernet connect during transferring, I see timeout error and the error msg: "Operation too slow. Less than **1250** bytes/sec transferred the last **30** seconds"
Does anybody know why?
And from the code, I could not fine where 1250 is set.

**Comment by bagder** [comment, marked as answer]
It is *not* set to 1250 by default in any version.

**Reply by CHENZENGHU** [reply]
Yes, my fault. I set the value in my own library.
